scripts/upload_cellguide_descriptions.py

```diff
@@ -0,0 +1,103 @@
+"""
+This is a script to upload cellguide descriptions using the /cellguide/v1/upload endpoint.
+
+It requires a csv file to be present in the scripts directory with the following columns:
+Cell Ontology ID, Final version (QC'd), Supporting reference 1, Supporting reference 2, Supporting reference 3, 
+Supporting reference 4, Supporting reference 5
+
+The script will convert the csv to json and post each row to the endpoint.
+Step 1 - Login: python upload_cellguide_descriptions.py --deployment staging login --api_key <api_key>
+Step 2 - Upload: python upload_cellguide_descriptions.py --deployment staging upload --path cl_descriptions.csv
+"""
+
+import csv
+import json
+import os
+
+import click
+import requests
+
+
+@click.group()
+@click.option("--deployment", default="test", show_default=True, help="The name of the deployment to target")
+@click.pass_context
+def cli(ctx, deployment):
+    """
+    Set deployment stage and api base url
+    """
+    os.environ["DEPLOYMENT_STAGE"] = deployment
+    api_base = {
+        "rdev": "https://pr-6426-backend.rdev.single-cell.czi.technology",
```

Comment:
rdev option is here for testing purposes - once reviewed and approved this line will be removed before merging.